packages/vue/src/components/organisms/SfTabs/SfTabs.vue

```diff
@@ -16,6 +17,18 @@ export default {
       type: Number,
       default: 1,
     },
+    tabMaxContentHeight: {
```

Comment:
@AdamPawlinski can you add docs description for all props?

Reply:
Of course, will be done.

Reply:
thanks 💚